RF24Network.cpp

```diff
@@ -1001,7 +1001,9 @@ bool ESBNetwork<radio_t>::write_to_pipe(uint16_t node, uint8_t pipe, bool multic
         ok = radio.txStandBy(txTimeout);
         radio.setAutoAck(0, 0);
     }
-
+    else if (!ok) {
```

Comment:
```suggestion
    else if (!ok  && !multicast) {
```
Per your OP, this change is dependent on auto-ack being enabled. And, just above this block is
https://github.com/nRF24/RF24Network/blob/f77c5fe7fae65344106727fa24a4890b75d57473/RF24Network.cpp#L994
Did I overlook something?

---
Soooo glad I wrote doc-strings for the private functions here. It helped a lot. ☺️ 
https://github.com/nRF24/RF24Network/blob/f77c5fe7fae65344106727fa24a4890b75d57473/RF24Network.h#L872-L874

Reply:
The `ok` var will never return false if multicasting is enabled (auto-ack disabled) when writing, so I believe the `else if (!ok) {` is sufficient.

Reply:
good point!